extensions/cms/src/cmsResource/tree/treeProvider.ts

```diff
@@ -35,18 +35,18 @@ export class CmsResourceTreeProvider implements TreeDataProvider<TreeNode>, ICms
 				// Call to collect all locally saved CMS servers
 				// to determine whether the system has been initialized.
 				let cmsConfig = this._appContext.apiWrapper.getConfiguration();
-				let cachedServers = cmsConfig ? cmsConfig.cmsServers : [];
+				let cachedServers = cmsConfig.servers ? cmsConfig.servers : [];
 				if (cachedServers && cachedServers.length > 0) {
 					let servers = [];
-					cachedServers.forEach((server) => {
+					cachedServers.forEach(async (server) => {
```

Comment:
Same, why async?

Reply:
Still want to know if this is necessary, it doesn't seem to be

---
In reply to: [285770494](https://github.com/microsoft/azuredatastudio/pull/5542#discussion_r285770494) [](ancestors = 285770494)